app/views/posts/index.html.erb

```diff
@@ -1 +1,11 @@
-<h1>This is the first post</h1>
\ No newline at end of file
+<p> This is the list of posts</p>
+
+ <%= render'users/user', user: @user %>
+ 
+<% @posts.each_with_index do |post, index| %>
+<p>Post #<%= index+1 %></p>
+<section >
+  <ul><%= render 'posts/post', post: post %></ul>
+
+</section>
+<% end %>
```

Comment:
**OPTIONAL** - I suggest that you add a link on any of your pages, but preferably the post_index page, to lead to the form page for adding a new post (new_user_post_path). Please fix this 🙏